Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.14.0a3-slim
```

Comment:
Using a pre-release version of Python (`3.14.0a3`) might not be stable. Consider using a stable version like `3.10.8-slim` as recommended in the checklist.

Reply:
Done